SONAR-netCDF4/docs/crr341.adoc

```diff
@@ -140,7 +140,7 @@ SONAR-netCDF4 is a data and metadata convention for the storage of data from act
 
 Datasets can be added to SONAR-netCDF4 files if they do not conflict with the SONAR-netCDF4 datasets. If such additions are potentially of use to other users of the file format, it is recommended that they be proposed for inclusion in this or additional convention specifications.
 
-Each SONAR-netCDF4 file is intended to store data from one sonar mounted on one platform. The storage of data from multiple sonars and multiple platforms in one SONAR-netCDF4 file is not in the scope of the convention.
+Each SONAR-netCDF4 file is intended to store data from one type of sonar mounted on one platform. The storage of data from different sonar types and multiple platforms in one SONAR-netCDF4 file is not in the scope of the convention.
```

Comment:
The format assumes one sonar per file, not one type of sonar (where a multi-channel echosounder is considered one sonar).